apps/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ endif()
 if(CURSE)
     add_executable(${CURSE} curse.cpp lasfile.hpp las.hpp las.cpp OGR.hpp OGR.cpp mmaplib.hpp pdal_util_export.hpp ProgramArgs.hpp Utils.cpp Utils.hpp )
     target_link_libraries(${CURSE} ${HEXER_LIB_NAME} ${HEXERBOOST_LIB_NAME})
+    if(NOT WIN32)
+        target_link_libraries(${CURSE} dl)
```

Comment:
I think we're supposed to use `CMAKE_DL_LIBS` https://cmake.org/cmake/help/latest/variable/CMAKE_DL_LIBS.html